CONTRIBUTING.md

```diff
@@ -202,6 +203,19 @@ Once you have `nix` installed, build and enter the clean development shell with:
 $ nix develop
 ```
 
+#### Loading nix dependencies with direnv
+
+If you want to use direnv to setup your environment with nix (instead of using a
+shell), you will need to add `use flake;` to your `.envrc`, and then running
+`direnv allow`:
+
+```sh
+echo "use flake;" >> .envrc && direnv allow
```

Comment:
I think the `.local-envrc` could be used for this without requiring dirtying the worktree?

Reply:
Ah right, that should be better! Thanks